modules/core/src/v2/coins/algo.ts

```diff
@@ -193,6 +232,57 @@ export class Algo extends BaseCoin {
     return [0, 1, 2];
   }
 
+  /**
+   * Builds a funds recovery transaction without BitGo
+   * @param params
+   * - userKey: [encrypted] Algo private key
+   * - backupKey: [encrypted] Algo private key, or public key if the private key is held by a KRS provider
+   * - walletPassphrase: necessary if one of the private keys is encrypted
+   * - bitgoKey : bitgo xpub of the wallet to recover funds from
+   * - krsProvider: necessary if backup key is held by KRS
+   * - recoveryDestination: target address to send recovered funds to
+   * @param callback
+   */
+  recover(params: RecoveryOptions, callback: NodeCallback<RecoveryTransaction>): Bluebird<RecoveryTransaction> {
+    return co(function*() {
+      // const userPrv = this.bitgo.decrypt({ input: params.userKey, passphrase: params.walletPassphrase });
```

Comment:
Why is all this commented out?

Reply:
its not ready for review yet - the commented-out stuff is pretty much just a template